jinja2/filters.py

```diff
@@ -705,7 +707,11 @@ def do_groupby(environment, value, attribute):
        attribute of another attribute.
     """
     expr = make_attrgetter(environment, attribute)
-    return sorted(map(_GroupTuple, groupby(sorted(value, key=expr), expr)))
+    if sort:
+        return sorted(map(_GroupTuple, groupby(sorted(value, key=expr), expr)))
```

Comment:
not sure about memory consumption. Will `sorted` every time create a copy of value?